website/static/js/osfHelpers.js

```diff
@@ -746,7 +746,7 @@ var confirmDangerousAction = function (options) {
                 className: 'btn-default'
             },
             success: {
-                label: 'Confirm',
+                label: options.confirmBtnLabel,
```

Comment:
@sloria Ready to another review! Thank you!

yes, we should follow the default option. Originally, I tried to override with default option, but it will override the callback function also. Now I figure it out. I can assign callback to input options also.